pkg/apis/crd/v1alpha1/types.go

```diff
@@ -354,3 +355,146 @@ type BGPPeer struct {
 	// a restart before deleting stale routes. The range of the value is from 1 to 3600, and the default value is 120.
 	GracefulRestartTimeSeconds *int32 `json:"gracefulRestartTimeSeconds,omitempty"`
 }
+
+type PodReference struct {
+	Namespace string `json:"namespace"`
+	Name      string `json:"name"`
+}
+
+// Source describes the source spec of the packetcapture.
+type Source struct {
+	// Pod is the source Pod.
+	Pod *PodReference `json:"pod,omitempty"`
+	// IP is the source IPv4 or IPv6 address.
+	IP *string `json:"ip,omitempty"`
+}
+
+// Destination describes the destination spec of the PacketCapture.
+type Destination struct {
+	// Pod is the destination Pod, exclusive with destination Service.
```

Comment:
you seem to have removed Service from this API version, so the comment is no longer valid
However, it is mutually exclusive with IP

We should also say that IP and Pod are mutually exclusive for the Source struct above

Reply:
ok